client/src/components/OnboardingParent/OnboardingParent.js

```diff
@@ -4,18 +4,16 @@ import Logo from "../Logo/Logo.js";
 import CarouselDots from "../CarouselDots/CarouselDots";
 
 const OnboardingParent = props => {
+  const hello = () => {
+    console.log("Hello world!");
+  };
```

Comment:
Change this function to something more semantic. For example, you could call it `handleStatusChange` and have it console log a string indicating that the status has changed.